<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="All">
     <attribute name="title">
      <string>All</string>
     </attribute>
     <widget class="QListWidget" name="AllList">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>67</y>
        <width>351</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="UsersSearch_4">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>37</y>
        <width>351</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="UsersSearch_3">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string> Search</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Users">
     <attribute name="title">
      <string>Users</string>
     </attribute>
     <widget class="QLineEdit" name="UsersSearch">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string> Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UsersSearch_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>37</y>
        <width>351</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>search</string>
      </property>
     </widget>
     <widget class="QListWidget" name="usersList">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>67</y>
        <width>351</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Groups">
     <attribute name="title">
      <string>Groups</string>
     </attribute>
     <widget class="QListWidget" name="GroupList">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>67</y>
        <width>351</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string> Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>37</y>
        <width>351</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string> Search</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Channels">
     <attribute name="title">
      <string>Channels</string>
     </attribute>
     <widget class="QListWidget" name="ChannelList">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>67</y>
        <width>351</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string> Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>37</y>
        <width>351</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string> Search</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>621</width>
      <height>591</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="nameFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>621</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="nameLabel">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>name</string>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="messageList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>621</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="messageTextEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>540</y>
       <width>581</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-top-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>540</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>send</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border-image: url(:/new/prefix1/neumorphic-rectangle-button-free-png.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
